### PR TITLE
jobs/openshift/release: Update workers-rhel7 periodics

### PR DESCRIPTION
**Pull request by mtnbikenc**
Periodics for 4.6 and 4.7 are being switched to `workers-rhel7`.

`workers-rhel7` should be informing.

### CODE REVIEW

File: core-services/release-controller/_releases/priv/release-ocp-4.6.json [hunk 1]
```diff
@@ -51,6 +51,13 @@
                 "name": "release-openshift-ocp-installer-e2e-aws-upi-4.6-priv"
             }
         },
+        "aws-workers-rhel7": {
+            "disabled": true,
```
Comment (stevekuznetsov) [inline core-services/release-controller/_releases/priv/release-ocp-4.6.json:55]: You said you wanted to be blocking, which is not what this is doing

File: core-services/release-controller/_releases/release-ocp-4.6.json [hunk 2]
```diff
@@ -130,6 +130,10 @@
     "proxy":{
       "optional":true,
       "prowJob":{"name":"periodic-ci-openshift-release-master-ocp-4.6-e2e-aws-proxy"}
+    },
+    "aws-workers-rhel7":{
+      "optional":true,
```
Comment (stevekuznetsov) [inline core-services/release-controller/_releases/release-ocp-4.6.json:135]: Also this

File: core-services/release-controller/_releases/priv/release-ocp-4.6.json (hunk 1 above)
Comment (mtnbikenc) [reply core-services/release-controller/_releases/priv/release-ocp-4.6.json:55]: These were all disabled, so I left it as is.

File: core-services/release-controller/_releases/release-ocp-4.6.json (hunk 2 above)
Comment (mtnbikenc) [reply core-services/release-controller/_releases/release-ocp-4.6.json]: Updated.

File: core-services/release-controller/_releases/priv/release-ocp-4.6.json (hunk 1 above)
Comment (mtnbikenc) [reply core-services/release-controller/_releases/priv/release-ocp-4.6.json:55]: We've decided that workers-rhel7 should be informing.

**Comment by mtnbikenc** [comment]
4.7 periodics are not expected to pass yet.

File: ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml [hunk 3]
```diff
@@ -1819,3 +1819,731 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
```
Comment (sdodson) [inline ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml:1822]: This is all generated? Seems like quite a few unrelated jobs added here. Just want to confirm these are all intentional.

Comment (mtnbikenc) [reply ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml:1822]: This is all generated from the addition of ci-operator/config/openshift/release/openshift-release-master__ocp-4.7.yaml.  Steve said I could add it.  Maybe it is premature?

**Review by droslean — APPROVED** [review]

**Pull request merged**